### PR TITLE
Contact & Msg expressions context

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
Yes `@step.urn.scheme` will solve the immediate need

**Comment by rowanseymour** [comment]
My bias is to adding the absolute minimum needed for now to the old world - make sure there is a straightforward migration path to the new world (don't forget to add this in goflow!). Thoughts @nicpottier ?

**Comment by norkans7** [comment]
Now the question is do we add that just to step or generally to msg context which is the one the step context use initially if we have a msg

File: temba/contacts/models.py [hunk 1]
```diff
@@ -2301,7 +2307,13 @@ def get_urn_context(self, org, scheme=None):
             return ""
 
         if org.is_anon:
-            return ContactURN.ANON_MASK
+            return {
```
Comment (norkans7) [inline temba/contacts/models.py]: This also allow @contact.tel.scheme on anon orgs to be similarly accessible

**Comment by norkans7** [comment]
Replaced by #2094

**Pull request closed without merging**